.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: "gomod"
+    directory: "/"
+    # Check for updates once a day
+    schedule:
+      interval: "daily"
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    # Check for updates once a week
+    schedule:
+      interval: "weekly"
```

Comment:
This whole section is useless, right? There are no workflows or actions defined in this repo, so I don't think there is anything to check here.

Reply:
Yes, I think I just copied from an existing repository so I will remove it here.